lib/google_drive.rb

```diff
@@ -86,7 +86,8 @@ def self.login_with_oauth(client_or_access_token, proxy = nil)
     # +path+ defaults to ENV["HOME"] + "/.ruby_google_drive.token".
     #
     # You can specify your own OAuth +client_id+ and +client_secret+. Otherwise the default one is used.
-    def self.saved_session(path = nil, proxy = nil, client_id = nil, client_secret = nil)
+    def self.saved_session(path = nil, proxy = nil)
```

Comment:
Please keep client_id and client_secret here, and use them instead of those in the config file when given. Otherwise it will break existing code using these parameters.